### DISCUSSION TITLE
Why won't apk install in BlueStacks?

### DISCUSSION THREAD

**Discussion by antbelogurov** [quasarframework/quasar#9917] [Quasar App Extensions] [answered]
Hi!
Please suggest.
I have a project on vue quasar
I want to create an apk application and run it through bluestacks
The device emulator works correctly.
1.start the project using the command quasar build -m cordova -T android
![2233](https://user-images.githubusercontent.com/63599667/124250875-f5df5e80-db2d-11eb-8696-1fbdec26313a.png)
The only warning is from Gradle about the version
![223333](https://user-images.githubusercontent.com/63599667/124250916-0099f380-db2e-11eb-8aff-6a5bfec62b8e.png)
Then I ran gradle --warning-mode all to understand the reasons. 
The result is on the screenshot above.

Next, I drop the created apk into bluestaks, but it gives an error The application is not installed, it looks like the package is damaged.
I don’t understand just what is worth paying attention to? why not install or compile incorrectly?

**Comment by ontwikkelfabriek** [comment]
@antbelogurov 

> Next, I drop the created apk into bluestaks

You cannot install **unsigned** apk's by sideloading on any android device.  
So you have to sign the apk or deploy it with `cordova run` to your `bluestaks device`.

**Comment by metalsadman** [comment, marked as answer]
@antbelogurov what @ontwikkelfabriek said, you can build a debug signed apk for testing with the `--debug` flag.

**Reply by antbelogurov** [reply]
> @antbelogurov what @ontwikkelfabriek said, you can build a debug signed apk for testing with the `--debug` flag.

thank you so much
